### PR TITLE
Trusty 1.2

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: roles/common/meta/main.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ dependencies:
         key_url: '{{ apt_repos.hwraid.key_url }}'
       - repo: 'deb {{ apt_repos.sensu.repo }} sensu main'
         key_url: '{{ apt_repos.sensu.key_url }}'
+
```
Comment (j2sol) [inline roles/common/meta/main.yml]: Why a space here but not a space between other roles? Just curious.

**Comment by j2sol** [comment]
This all looks good to me.

File: envs/vagrant/defaults.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ country_code: US
 
 #openstack_install_method: 'source'
 openstack_install_method: 'package'
-openstack_package_version: '10.0-bbc62'
+openstack_package_version: '10.0-bbc72'
```
Comment (j2sol) [inline envs/vagrant/defaults.yml]: This needs to be updated in the example environment too.

**Comment by j2sol** [comment]
msg: No package matching 'openstack-keystone-10.0-bbc62' is available

**Comment by paulczar** [comment]
retest

**Comment by j2sol** [comment]
So this needs to be rebased I think.

**Pull request merged**